docs/circleci.md

```diff
@@ -0,0 +1,58 @@
+---
+id: circleci
+title: CircleCI
+---
+
+Lets start with the basics, [CircleCI](https://circleci.com) is the service we use in order to run our tests and to check the integrity of our code.
+
+As you are probably aware, while pushing any code to our repos it will initiate a build within CircleCI, this build will execute several things, or in the Circle language steps.
+Lets go over our steps to understand what each of them is doing:
+- Prepare Environment:
+    - Sets up the testing environment before starting the build. This usually involves provisioning a test server, setting access rules, etc.
+- Install dependencies:
+    - Here we install our python packages, and give access to the scripts folder. The scripts folder contains all of the scripts we are using for the next steps in Circle.
+- Validate Files and Yaml
+    - Validates the schema of the yml files you created, and checks to ensure that you haven’t made any changes that may effect backwards compatibility.
+    - [You can learn more about the YAML structure here](/docs/yaml-file-integration)
```

Comment:
Broken link. I believe it should be `/docs/yaml-file`

Reply:
will fix